user/templates/windows/qubes-windows-tools-4-1.md

```diff
@@ -173,7 +173,7 @@ Installing the Qubes Windows Tools on Windows 7, 8.1, 10 and 11 both as a Standa
 	
 	 If Windows is used in a TemplateVM / AppVM combination, this registry fix has to be applied to the TemplateVM, as the `HKLM` registry key belongs to the template-based part of the registry.
 	
- 10. Lastly to enable file copy operations to a Windows VM, the `default_user` property of this VM should be set to the `<username>` that you use to login to the Windows VM. This can be done via the following command on a `dom0` terminal: *(where `<VMname>` is the name of your Windows VM)*
+ 10. Lastly to enable file copy operations to a Windows VM, the `default_user` property of this VM should be set to the `<username>` that you use to login to the Windows VM. This can be done via the following command on a `dom0` terminal: *(where <VMname> is the name of your Windows VM)*
```

Comment:
Is this correct? I think without quoting this will be interpreted as (invalid) html tag... (but I haven't checked)
Similar comments below

Reply:
This is because rst cannot embed emphasis inside italics and vice versa etc, as far as I read the documentation. I can leave it unchanged but it will be broken after conversion. You are right about the html tag.
Curiously enough when I rendered sphinx locally there was not a problem.
Here with the original markdown: https://current-qubes-docrtd.readthedocs.io/en/rtd-deploy-test/user/templates/windows/qubes-windows-tools-4-1.html vs the fix from above: https://current-qubes-docrtd.readthedocs.io/en/rtd-deploy-pr/user/templates/windows/qubes-windows-tools-4-1.html. Search for 'command in dom0'
I will revert it and it can be fixed after conversion